examples/example.ipynb

```
@@ -0,0 +1,430 @@
+                "\n",
+                "\n",
+                "class Library(models.Model):\n",
+                "    name = models.CharField(max_length=100)\n",
+                "    address = models.TextField()\n",
+                "\n",
+                "    class Meta:\n",
+                "        app_label = \"tests\"\n",
+                "\n",
+                "\n",
+                "class BookCopy(models.Model):\n",
+                "    book = models.ForeignKey(Book, on_delete=models.CASCADE)  # Foreign Key relationship\n",
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)